.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
+    "es2015",
```

Comment:
It appeared to knacker module imports in `server.js`, which I meant to rewrite, actually... add it back in if I did it wrong